src/schnetpack/representation/painn.py

```diff
@@ -161,8 +167,28 @@ def __init__(
         self.cutoff_fn = cutoff_fn
         self.cutoff = cutoff_fn.cutoff
         self.radial_basis = radial_basis
+        self.activate_charge_spin_embedding = activate_charge_spin_embedding
 
-        self.embedding = nn.Embedding(max_z, n_atom_basis, padding_idx=0)
+        # nuclear embedding layer (often complex nuclear embedding has negative impact on the performance of the model, so we can use simple nuclear embedding to avoid this issue)
+        if nuclear_embedding != "simple":
```

Comment:
I agree. I wanted to say the same in this comment: https://github.com/atomistic-machine-learning/schnetpack/pull/608#discussion_r1506256746

Reply:
Ah True. didn`t see this

Reply:
done in c86b404 and f99a432
please let me know if this is sufficient to circumvent the clunky if calls to decide which nuclear embedding to use.

Reply:
looks fine to me, but we need to add `self.nuclear_embedding = nuclear_embedding` before the if-statement. Otherwise the embedding is missing if it is not `None`

Reply:
done in c503f6b